In [31]:
nome = "exagerado_letra.txt"
letra = input("Escolha uma letra para buscar no texto: ")

arquivo = open(nome, "r", encoding="utf-8")

conteudo = arquivo.read()

conteudo = conteudo.lower()

cont = conteudo.count(letra.lower())

print(f"Vezes em que a letra {letra.upper()} apareceu na música: {cont}")

arquivo.close()

Vezes em que a letra A apareceu na música: 110


In [27]:
nome = "exagerado_letra.txt"
letra = input("Escolha uma letra para buscar no texto: ")

arquivo = open(nome, "r", encoding="utf-8")

conteudo = arquivo.read()

conteudo = conteudo.lower()

cont = conteudo.count(letra.lower())

print(f"Vezes em que a letra {letra.upper()} apareceu na música: {cont}")

arquivo.close()

Vezes em que a letra E apareceu na música: 64


In [23]:
nome = "exagerado_letra.txt"
letra = input("Escolha uma letra para buscar no texto: ")

arquivo = open(nome, "r", encoding="utf-8")

conteudo = arquivo.read()

conteudo = conteudo.lower()

cont = conteudo.count(letra.lower())

print(f"Vezes em que a letra {letra.upper()} apareceu na música: {cont}")

arquivo.close()

Vezes em que a letra É apareceu na música: 11


In [25]:
nome = "exagerado_letra.txt"
letra = input("Escolha uma letra para buscar no texto: ")

arquivo = open(nome, "r", encoding="utf-8")

conteudo = arquivo.read()

conteudo = conteudo.lower()

cont = conteudo.count(letra.lower())

print(f"Vezes em que a letra {letra.upper()} apareceu na música: {cont}")

arquivo.close()

Vezes em que a letra Ê apareceu na música: 4


In [28]:
nome = "exagerado_letra.txt"
letra = input("Escolha uma letra para buscar no texto: ")

arquivo = open(nome, "r", encoding="utf-8")

conteudo = arquivo.read()

conteudo = conteudo.lower()

cont = conteudo.count(letra.lower())

print(f"Vezes em que a letra {letra.upper()} apareceu na música: {cont}")

arquivo.close()

Vezes em que a letra I apareceu na música: 33


In [29]:
nome = "exagerado_letra.txt"
letra = input("Escolha uma letra para buscar no texto: ")

arquivo = open(nome, "r", encoding="utf-8")

conteudo = arquivo.read()

conteudo = conteudo.lower()

cont = conteudo.count(letra.lower())

print(f"Vezes em que a letra {letra.upper()} apareceu na música: {cont}")

arquivo.close()

Vezes em que a letra O apareceu na música: 92


In [30]:
nome = "exagerado_letra.txt"
letra = input("Escolha uma letra para buscar no texto: ")

arquivo = open(nome, "r", encoding="utf-8")

conteudo = arquivo.read()

conteudo = conteudo.lower()

cont = conteudo.count(letra.lower())

print(f"Vezes em que a letra {letra.upper()} apareceu na música: {cont}")

arquivo.close()

Vezes em que a letra U apareceu na música: 41


Ao analisar as vogais que existem na música, podemos tirar como conclusão que existem na mesma: 

Vezes em que a letra A apareceu: 110

Vezes em que a letra E apareceu: 64

Vezes em que a letra É apareceu: 11

Vezes em que a letra Ê apareceu: 4

Vezes em que a letra I apareceu: 33

Vezes em que a letra O apareceu: 92

Vezes em que a letra U apareceu: 41
